Makefile

```diff
@@ -115,7 +115,7 @@ SRC:=
 all : buildall
 
 # Set up basic nvcc options and add CUDA targets from above
-CUFLAGS = -m 64
+CUFLAGS = -m 64 -std=c++11
```

Comment:
This works. But is it because NVCC does not support c++14 (?)

Reply:
According to the docs, Cuda 9 supports c++14 but Cuda 8 only supports c++11. I believe CNTK still requires Cuda 8, not 9, but when that changes we could make the standard uniform again.